client/cypress/integration/InfoBase/route_tests.spec.js

```diff
@@ -247,38 +247,37 @@ const route_load_tests_config = [
 
 describe("Route tests", () => {
   route_load_tests_config.map((routes) => {
-    describe(`${routes.name} route`, () => {
+    describe(`${routes.name}`, () => {
       routes.test_on.map((app) => {
-        it(`Tested on ${app}`, () => {
-          cy.on("fail", (e, test) => {
-            if (routes.expect_to_fail) {
-              console.log("Test expected to fail.");
-            } else {
-              throw e;
-            }
-          });
-
+        it(`Tested on index-${app}.html#${routes.route}`, () => {
           cy.visit(
             `http://localhost:8080/build/InfoBase/index-${app}.html#${routes.route}`
           );
-          if (!routes.skipAxe) {
-            if (app == "eng" || app == "fra") {
-              cy.injectAxe();
-              cy.get(".leaf-spinner__inner-circle", { timeout: 10000 }).should(
-                "not.exist"
-              );
-              cy.checkA11y(
-                null,
-                { includedImpacts: ["critical"] },
-                cy.terminalLog,
-                false
-              );
-            }
+
+          if (!routes.skipAxe && (app == "eng" || app == "fra")) {
+            cy.injectAxe();
+            cy.get(".leaf-spinner__inner-circle", { timeout: 10000 }).should(
+              "not.exist"
+            );
+            cy.checkA11y(
+              null,
+              { includedImpacts: ["critical"] },
+              cy.terminalLog,
+              false
+            );
           } else {
             cy.get(".leaf-spinner__inner-circle", { timeout: 10000 }).should(
               "not.exist"
             );
           }
+
+          cy.on("fail", (e, test) => {
+            if (routes.expect_to_fail) {
+              console.log("Test expected to fail.");
```

Comment:
Might be worth logging what it failed with (not re-throwing, just as a log). If those always-failing routes are still flaky then that'd help us figure out why